### DISCUSSION TITLE
Support for Multi-Word Search

### DISCUSSION THREAD

**Discussion by niemyjski** [home-assistant/frontend#7810] [Dashboard]
Currently it's impossible to quickly filter automations by more than one term.

![image](https://user-images.githubusercontent.com/1020579/100218337-3ca31100-2eda-11eb-8a83-4a38c43fe04f.png)

![image](https://user-images.githubusercontent.com/1020579/100218358-4593e280-2eda-11eb-8e19-62220eb5a704.png)

It would be super useful to be able to match on the following criteria:

1. Use the current behavior.
2. Split on white space and special characters and match on included terms.

**Comment by spacegaier** [comment]
Perhaps a fuzzy search as used in the quick bar, could make sense here?

**Reply by niemyjski** [reply]
That would work. I'd love to have consistent search experience across the app
